packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-hierarchy.md

```diff
@@ -12,13 +12,15 @@ This Pattern shows an expantable table of contents which can contain simple link
 ### Usage Guidelines
 * This is meant to be used to contain a mix of links, download links, or section titles with child links/download links within accordions.
 * `isCurrent` flags are only applied when this organism is used within the table-of-contents-overlay organism. Additionally, `isCurrent` should be applied to both the top level of an item containing `linkItems`, as well as the item within `linkItems`.
-
+* Use `background: "none"` for the no background variant.
```

Comment:
If background "none" is needed I wonder if this works on OpenMass. I mean, that the Table of Contents Hierarchy receives the correct value for no background, instead of an empty string, NULL or similar. Do you know the steps to test this?

Reply:
I think it needs to be specified as "none", see this line https://github.com/massgov/mayflower/pull/1589/files/cd85179bfc9b28ca2e9d9f935ab75caa081b5f5d#diff-2e42a4d1cfe2b31683b2b7bb74bac67a549078542419041bc53481b3374a0df1R1 
I believe by default, or without the "none" value, it uses the default style which is the one with a light gray background that's used on a binder page TOC section.  Javier added this variant in his original work, I copied it over. Let me know if that answers your question